build.gradle

```diff
@@ -130,9 +170,11 @@ task syncVersion {
 
 // Clean all
 task clean(type: Delete) {
-    delete "$buildDir"
+    doLast() {
+        delete "$buildDir"
 
-    delete fileTree("$componentDir") { include "*.xam" }
+        delete fileTree("$componentDir") { include "*.xam" }
+    }
```

Comment:
The delete commands need to be in a `doLast()`, otherwise they run every time, whether or not `clean` is requested. This error was repeated in several `build.gradle` files.